Omikron/Factfinder/Model/Export/Product.php

```diff
@@ -194,7 +194,7 @@ public function exportProducts($updateFieldRoles = false)
 
         foreach ($stores as $store) {
             $currChannel = $this->helperData->getChannel($store->getId());
-            if (in_array($currChannel, $exportedChannels)) {
+            if (in_array($currChannel, $exportedChannels) || !$this->helperData->isEnabled($store->getId())) {
```

Comment:
It would be helpful if the module is capable to export the data in disabled state because there might be some agencies or customers which don't have a testing environment.